dominate/tags.py

```diff
@@ -754,6 +754,68 @@ class area(html_tag):
 
 
 
+# SVG
```

Comment:
I'd put this in a separate file/namespace and not with the rest of the HTML tags

Reply:
Moved svg tags to their own file -- as with the tests.